app/vmctl/flags.go

```diff
@@ -728,6 +728,32 @@ var (
 	}
 )
 
+const (
+	backoffRetries     = "backoff-retries"
+	backoffFactor      = "backoff-factor"
+	backoffMinDuration = "backoff-min-duration"
+)
+
+var (
+	backoffFlags = []cli.Flag{
+		&cli.IntFlag{
+			Name:  backoffRetries,
+			Value: 10,
+			Usage: "How many retries to perform before giving up. Default is 10.",
```

Comment:
```suggestion
			Usage: "How many retries to perform before giving up.",
```
Default will be printed by flags library, no need to put this into a description manually.

Reply:
fixed